### PR TITLE
Cia

### PR DESCRIPTION
**Pull request by zshaheen**
* First take at the Common Input Arguments ([CIA](https://cia.gov))

### CODE REVIEW

File: example/example_parser_cia.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import cdp
+
+class MyCDPParameter(cdp.cdp_parameter.CDPParameter):
+     def check_values(self):
+          pass
+
+
+parser = cdp.cdp_parser.CDPParser(MyCDPParameter(),["/export/gleckler1/git/CIA/Defaults/DefArgsCIA.json",])
```
Comment (durack1) [inline example/example_parser_cia.py]: This hard coded (and machine dependent) path is a BAAAAAD idea

File: share/default_args.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        "dest":"other_parameters",
+        "default":[],
+        "help":"Path to other user-defined parameter file",
+        "required":false},
+        "--num_workers": {"aliases":["-n"],
+            "type":"int",
+            "help":"Number of workers, used when runnnig with multiprocessing or distributedly",
+            "required":false},
+            "--scheduler_addr":{"type":"str",
+                "help":"Address of scheduler in the form of IP_ADRESS:PORT. Used when runnnig distributedly"}}
```
Comment (durack1) [inline share/default_args.json]: Might want to fix `runnnig` -> `running`, and probably better "Used when running in distributed mode" - same for above, line 14 comment

File: example/example_parser_cia.py (hunk 1 above)
Comment (zshaheen) [reply example/example_parser_cia.py]: Thanks, we'll fix this. @doutriaux1

File: share/default_args.json (hunk 2 above)
Comment (zshaheen) [reply share/default_args.json]: I'll fix this too. Thanks for catching this stuff.

Comment (durack1) [reply share/default_args.json]: No problem, you probably want to also correct "IP_ADRESS:PORT" -> "IP_ADDRESS:PORT"

File: share/default_args.json [hunk 3]
```diff
@@ -14,4 +14,4 @@
             "help":"Number of workers, used when runnnig with multiprocessing or distributedly",
```
Comment (durack1) [inline share/default_args.json]: Here too!

File: cdp/cdp_parser.py [hunk 4]
```diff
@@ -255,27 +255,27 @@ def load_default_args(self, files=[]):
...
             default=[],
-            help='Path to the other user-defined parameter file',
+            help='Path to the other user-defined parameter file.',
             required=False)
         self.add_argument(
             '-n', '--num_workers',
             type=int,
             dest='num_workers',
-            help='Number of workers, used when running with multiprocessing or distributedly',
+            help='Number of workers, used when running with multiprocessing or distributedly.',
```
Comment (durack1) [inline cdp/cdp_parser.py]: @zshaheen as far as my english stretches, `distributedly` isn't a word. Might I suggest taking the edit below "Used when running in distributed mode" and also applying it here :)

File: share/default_args.json [hunk 5]
```diff
@@ -11,7 +11,7 @@
         "required":false},
         "--num_workers": {"aliases":["-n"],
             "type":"int",
-            "help":"Number of workers, used when runnnig with multiprocessing or distributedly",
+            "help":"Number of workers, used when running with multiprocessing or distributedly",
```
Comment (durack1) [inline share/default_args.json]: Same here

File: cdp/cdp_parser.py (hunk 4 above)
Comment (durack1) [reply cdp/cdp_parser.py]: @zshaheen it seems I'm wrong (https://www.merriam-webster.com/dictionary/distributedly) but I still think my text is clearer

Comment (zshaheen) [reply cdp/cdp_parser.py]: Okay, done in 1a7cfc3dc4899bfd3c43c5f72e9484912535aced

**Pull request merged**